client/api/notebook.py

```diff
@@ -97,6 +101,21 @@ def save_notebook(self):
         else:
             print()
 
+def login_with_env(assignment):
+    access_token = os.environ.get('OKPY_ACCESS_TOKEN')
+    if not access_token:
+        log.info("ACCESS_TOKEN did not exist in the environment")
+        return
+    student_email = ok_auth.display_student_email(assignment.cmd_args, access_token)
+    if student_email:
+        # Token is valid,
+        expires_in = int(os.environ.get('OKPY_EXPIRES_IN', 60))
+        refresh_token = os.environ.get('OKPY_REFRESH_TOKEN')
```

Comment:
> Is there a danger in just using the OKPY_ACCESS_TOKEN and not relying on the ok storage? Part of the reason why I'm stuck merging this is I'm not sure how to address the OKPY_ACCESS_TOKEN-storage token potential conflict

I think it's reasonable to merge as is - the first time the user instantiates the notebook api, we'll grab the env var and see if it's valid. If it is, we store it in `.ok_storage`. I'm not sure I see the conflict (other than the case where one is valid for longer than the other) 

If the user somehow had a longer lasting token in `.ok_storage` and the new token we set ends up expiring - it's not that bad of an outcome. When they try to submit, we'll just reprompt to login. We could do what Kyle suggested - check which token is valid for longer and prefer that. I think it'll be fine even if you don't.

BTW - I can't imagine that many jupyter notebook sessions will contain expired access tokens since we get a new one every time they login. Sessions on the notebook server timeout/expire after a few hours of inactivity so students will have to login again.